pymc3/distributions/timeseries.py

```diff
@@ -206,15 +213,46 @@ def logp(self, x):
         -------
         TensorVariable
         """
-        sigma = self.sigma
-        mu = self.mu
-        init = self.init
+        if x.ndim > 0:
+            x_im1 = x[:-1]
+            x_i = x[1:]
+            if self.sigma.ndim > 0:
+                sigma = self.sigma[:-1]
```

Comment:
This contradicts the docstring that says that the first element is discarded, not the last

Reply:
oof! thanks for catching